Sources/Fuzzilli/Protobuf/ProtoUtils.swift

```diff
@@ -64,3 +66,56 @@ class OperationCache {
         operations.append(operation)
     }
 }
+
+public func decodeProtobufCorpus(_ buffer: Data) throws -> [Program]{
+    let opCache = OperationCache.forDecoding()
+    var offset = 0
+    
+    var newPrograms = [Program]()
+    while offset + 4 < buffer.count {
+        let value = buffer.withUnsafeBytes { $0.load(fromByteOffset: offset, as: UInt32.self) }
+        let size = Int(UInt32(littleEndian: value))
+        offset += 4
+        guard offset + size <= buffer.count else {
+            throw FuzzilliError.corpusImportError("Invalid program size in corpus")
+        }
+        let data = buffer.subdata(in: offset..<offset+size)
+        offset += size + align(size, to: 4)
+        let proto = try Fuzzilli_Protobuf_Program(serializedData: data)
+        let program = try Program(from: proto, with: opCache)
+        newPrograms.append(program)
+    }
+    return newPrograms
+}
+
+public func encodeProtobufCorpus<T: Collection>(_ programs: T, _ logger: Logger!) -> Data where T.Element == Program {
```

Comment:
This function requiring a `Logger` instance probably prevents its reuse in some cases. I think it'd be fine to either just drop the logging entirely or alternatively to have this function also return how many programs it successfully serialized, then the caller can still log a message based on that.